src/test/java/com/emc/object/s3/S3JerseyClientTest.java

```diff
@@ -2278,6 +2275,23 @@ public void testPreSignedUrlHeaderOverrides() throws Exception {
         Assert.assertEquals(contentDisposition, response.getHeaders().getFirst(RestUtil.HEADER_CONTENT_DISPOSITION));
     }
 
+    @Test
+    public void testVPoolHeader() {
+        CreateBucketRequest createBucketRequest = new CreateBucketRequest("looney-bucket")
+                .withVPoolId("plylab-sp1");
+        client.createBucket(createBucketRequest);
```

Comment:
please make sure you delete the bucket at the end of the test.. unit tests should always clean up any resources they create

Reply:
✔️